admin/views/partials/_print_pupils.ejs

```diff
@@ -26,10 +26,10 @@
                             <% } %>
                             <div class="pupil-content">
                             <span class="fullname"><%=
-                                    `${pupils[i].foreName} ${pupils[i].showMiddleNames
+                                    `${pupils[i].foreNameAlias || pupils[i].foreName} ${pupils[i].showMiddleNames
```

Comment:
I would prefer to have a simple field printed to screen here, populated by the presentation layer.

We only have two views, a pupil view of the name, and the staff view.

Reply:
There is no requirement for the staff view to display anything else rather than the actual names.
I added a pupil name presenter to capture the pupil view level.